engines/ospray/ispc/geometry/Cylinders.ispc

```diff
@@ -64,9 +63,14 @@ unmasked void Cylinders_bounds(const RTCBoundsFunctionArguments* uniform args)
              : self->radius;
     uniform vec3f v0 = *((vec3f * uniform)(cylinderPtr + self->offset_v0));
     uniform vec3f v1 = *((vec3f * uniform)(cylinderPtr + self->offset_v1));
-    // TODO less conservative bounds
+
+    uniform vec3f a2 = sqr(v0 - v1);
```

Comment:
I cannot figure out what you are calculating here, can you name these variables to something meaningful?

Reply:
I with I could understand this myself.